### DISCUSSION TITLE
Problema heroku del desarrollo del Addon de CguZZman

### DISCUSSION THREAD

**Discussion by Srkamber** [cguZZman/repository.plugins#16] [Ideas]
El addon google Drive ha dejado de funcionar.  Ahora mismo si entras a través de la autentificacion de https://drive-login.herokuapp.com/ te arroja un error, heroku dejo de ser gratis el 28 de noviembre, así que habrá que hacer entre toda la comunidad el pago de dicho servicio donde está el addon alojado.

**Comment by cguZZman** [comment]
Heroku removed the free tier server service. I'm looking for a new service. It will be down in the meantime.

**Comment by cguZZman** [comment]
Read my full response here: https://forum.kodi.tv/showthread.php?tid=324784&pid=3120043#pid3120043

**For now on, the server will be up as long as the community donates and help with the cost of the hosting.** If you are using the addon, don't be shy - any amount will help.
